src/runtime.js

```diff
@@ -22,6 +27,15 @@ let tornado = {
   },
 
   /**
+   * Register a helper, overwrite if a helper already exists
+   * @param {String} name The name of the helper
+   * @param {Function} method The function to be executed when the helper is found in a template
+   */
+  registerHelper(name, method) {
```

Comment:
No need to have an internal and client helpers object. My worry is squashing the helpers.  I believe one problem in the past was contained in this issue:

https://github.com/linkedin/dustjs-helpers/issues/72